### DISCUSSION TITLE
Does the new save format open up the option for new park awards?

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#16135] [Feature requests]
I'm sure we're all familiar with [the existing park awards](https://rct.fandom.com/wiki/The_Complete_Awards_List).

Now that we have a new save format, does that make it possible to have _new_ park awards? (Or old rewards applied in new ways?)

Some ideas about what a few new awards could be (and I'm sure others here could come up with many more) --

- 'Most Dangerous Park in the Country' -- there has been a high number of guest fatalities in the past year
- 'Most Improved' -- park rating went from very low to very high within the past year
- 'Dying Park Award' -- park rating went from very high to very low within the past year
- 'Best Transportation System' -- over a certain number of transport rides + over a certain percentage of guests using them
- 'Best Value Park' -- same as currently existing, but also applied to pay-per-ride parks if a high enough percentage of guests are thinking '[ride] is a really good value'
- 'Fastest Growing Park' -- A large number of rides built in the past year + a large increase in guests in park in the past year
- 'Most Innovative Park' -- R&D funding has been set on maximum for at least 2 years + at least 1/2 of newly developed ride types in the past year have been built
- 'Historical Park' -- Park contains at least 5 rides over 10 years old + those rides have high popularity levels with guests
- 'Most Overrated Park' -- A high amount of advertising + low park rating
- 'Undiscovered Gem' -- No advertising in park's history + high park rating + fewer than 1000 guests
- 'Most Overpriced Park' -- Park entrance fee too high and/or a high percentage of guests thinking 'I'm not paying that much for ____'
- 'Best Park for Children' -- Over 70% of rides have low intensity + at least 1 entertainer per 100 guests + at least 10 rides
- 'Most Thrilling Park' -- Over 70% of rides have high intensity + over 70% of rides have high excitement + at least 10 rides
- 'Most Dizzying Park' -- Over 50% of rides have high nausea + at least 10 rides
- 'Mister Bones' Stamp of Approval' -- At least 1 ride with a ride duration of over 30 minutes.
- 'Best Freebies' -- at least 3 shops/stalls in the park giving items away for $0 + those shops/stalls have 'sold' at least 300 items in the past month
- 'Best All-Day Park' -- over 70% of guests have been in the park for more than 3 in-game years
- 'Best Park Music' -- Over 80% of guests are close enough to a music-playing ride to hear the music
- 'Best Umbrellas/Balloons/T-shirts/etc...' -- Over 80% of guests are carrying the specified item
- Best Park to Work For' -- 100% of park staff have customized names + no park staff have been fired in the last year
- 'Friendliest Management' -- At least 5 guests have customized names + at least 1 guest thinking 'I have the strangest feeling someone is watching me' + at least 5 guests picked up and placed with tweezers in the past year + at least 5 guests have notifications enabled
- 'Most Decrepit Park' -- Over 25% of rides are broken down
- 'Longest Lines' -- Over 40% of guests are currently queuing for rides + at least 300 guests
- 'Shortest Lines' -- Under 10% of guests are currently queuing for rides + at least 10 rides in park + park at least 2 years old
- 'Wildlife Refuge' -- At least 100 ducks in park + at least 500 trees in park (both only counting land owned by park) + clicked at least one duck in the past year
- 'Ecological Disaster' -- A high number of released balloons + scenario not recently won (to avoid triggering it when peeps clap) + a high amount of litter (not pukes) on paths
- 'Heroic Staff Award' -- A station brakes failure occurs on a ride with more than 1 train, but a mechanic repairs the ride before any crash occurs.
- 'Arbor Foundation Award' -- Park has at least 20 trees per 100 tiles of owned land + no trees have been demolished in the past year

Anyway ... it's something to think about, maybe? Personally, I think getting awards is one of the coolest and most fun parts of the game, especially for players who don't know all the ins and outs of how the awards are calculated, so they get to discover new awards they weren't expecting and perhaps hear about amazing new awards.

Of course, you might not want to add as many ideas as I just came up with ... much less as many as others might add. But hopefully there are a few award ideas in that list that inspire enough to consider adding new park awards?

**Comment by James103** [comment]
* 'Best Park Overall' -- Park meets the criteria for a lot of positive awards and little to no negative awards.
* 'Worst Park Overall' -- Park meets the criteria for a lot of negative awards and few positive awards.
* 'Best Trees Award' -- Alternative title to 'Arbor Foundation Award'. Maybe add "Park has at least X different types of trees"?
* 'Longest Lines' & 'Shortest Lines' - Maybe add average queue time requirements?
* 'Best Freebies' - At least 3 **different types of** shops/stalls in the park are giving items away for $0.

**Comment by frutiemax** [comment]
This list of possible awards should probably be expended manually/loaded from file and saved into the park file. I don't think it's a good idea to hardcode new strings for awards. Scripts could be used to trigger the custom awards.

**Comment by J-PIE-314** [comment]
This PR looks to add plugin API for custom awards: https://github.com/OpenRCT2/OpenRCT2/pull/24468

**Reply by J-PIE-314** [reply]
This PR has been merged.
